cypress/e2e/comments.spec.ts

```diff
@@ -28,17 +30,33 @@ const MOCK_COMMENT = "mock comment"
 const ignoreResizeError = () =>
   cy.on("uncaught:exception", (err) => !err.message.includes("ResizeObserver"))
 
+const checkCommentVisible = (
+  commentText: string,
+  userEmail: string
+): Cypress.Chainable<JQuery<HTMLElement>> => {
+  return cy
+    .contains("div", commentText)
+    .should("be.visible")
+    .parent()
+    .contains(userEmail)
+    .should("be.visible")
+}
+
+const checkCommentDisabled = () =>
+  cy.get(COMMENTS_DRAWER_BUTTON_SELECTOR).should("be.disabled")
+
 const USER_NOT_AUTHORISED_COMMENT_ERROR_MESSAGE =
   "Only collaborators of a site can view review request comments!"
+const NOTIFICATION_CANNOT_BE_RETRIVED_ERROR_MESSAGE =
+  "Your notifications could not be retrieved. Please try again or check your internet connection"
 
-const submitComment = () => cy.get(SUBMIT_COMMENT_BUTTON_SELECTOR).click()
 const COMMENT_INTERCEPTOR = "getComments"
 
 describe("Comments", () => {
   beforeEach(() => {
     cy.setEmailSessionDefaults("Email admin")
+    cy.intercept("GET", "**/comments").as(COMMENT_INTERCEPTOR)
     cy.setupDefaultInterceptors()
-    visitE2eEmailTestRepo()
```

Comment:
why was this line deleted?

Reply:
Doesn't affect tests because most of them visit either this page or other pages on set up. I can add it back